recipes/caper/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "caper" %}
+{% set version = "1.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 33a963c47f9b43c6070c81cd80a40b27df3c00ccc1f7078045a84bb0a7ea8f59 
+
+build:
+  skip: True  # [py<36 or win]
```

Comment:
Can you go back to `noarch: python` here? We can ignore the windows build failure for that case. I'm assuming it fails due to a package being unavailable on windows?

Reply:
This package is not supposed to work on Windows. I added `noarch: python` back but got linting errors. How can we ignore Windows build?

Reply:
Oh. Well, if it's outright broken on Windows, then it shouldn't be noarch and you can take that back out. Apologies for the confusion.

Reply:
@dopplershift: Thanks, I just removed the `noarch` stuff.